_posts/2019-03-27-pulp-3-rc-release.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Pulp 3 Core RC Release
+author: David Davis
+---
+
+We are very excited today to announce the RC release of pulpcore and pulpcore-plugin packages. These
+packages include the core of Pulp 3 and its plugin API, and are now available on PyPI. They include
+pulpcore 3.0.0rc1 and pulpcore-plugin 0.1.0rc1.
+
```

Comment:
Does it make sense to identify they can use pulp_file right away? I also like the derivative benefit of setting the expectation that users will get plugin documentation and instructions separate from the core docs.

Reply:
I was thinking the pulp_file release would be a separate announcement which would sort of give other plugins a protocol to follow. Also, I don't feel like we should conflate the two releases (pulpcore and pulp_file) to avoid creating the expectation that one release is always tied to the other.

Reply:
@daviddavis that makes sense to me. I really like this blog post.